components/nginx/scripts/start.py

```diff
@@ -8,8 +8,12 @@
     join('components', 'utils.py'),
     join(dirname(__file__), 'utils.py'))
 import utils  # NOQA
-NGINX_SERVICE_NAME = 'nginx'
 
+NGINX_SERVICE_NAME = 'nginx'
 
 ctx.logger.info('Starting Nginx Service...')
 utils.start_service(NGINX_SERVICE_NAME, append_prefix=False)
+utils.systemd.verify_alive(NGINX_SERVICE_NAME, append_prefix=False)
+
+nginx_url = 'http://127.0.0.1'
```

Comment:
Let's try and understand if we're using SSL here or not. If https is enabled, we should probably check 443 instead.